frame/pablo/rpc/src/lib.rs

```diff
@@ -118,4 +126,25 @@ where
 			)))
 		})
 	}
+
+	fn redeemable_assets_for_given_lp_tokens(
+		&self,
+		pool_id: SafeRpcWrapper<PoolId>,
+		lp_amount: SafeRpcWrapper<Balance>,
+		at: Option<<Block as BlockT>::Hash>,
+	) -> RpcResult<RedeemableAssets<SafeRpcWrapper<Balance>>> {
+		let api = self.client.runtime_api();
+
+		let at = BlockId::hash(at.unwrap_or_else(|| self.client.info().best_hash));
+
+		// calling ../../runtime-api
+		let runtime_api_result = api.redeemable_assets_for_given_lp_tokens(&at, pool_id, lp_amount);
+		runtime_api_result.map_err(|e| {
+			RpcError::Call(CallError::Custom(ErrorObject::owned(
+				9876,
+				"Something wrong",
+				Some(format!("{:?}", e)),
```

Comment:
if other rpc do string errors i am ok. just confirm?

Reply:
This is something that I have to review still, every rpcs errors are like this right now 👍